compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/test/insert_slice.mlir

```diff
@@ -68,3 +69,42 @@ func.func @insert_slice_within_dispatch_workgroups_not_converted() -> tensor<f32
   }
   return %0 : tensor<f32>
 }
+
+// -----
+
+func.func @insert_slice_convert_dynamic_offset_and_size
+    (%arg0 : tensor<?x24x48xf32>, %arg1 : tensor<1x?x48xf32>, %offset : index, %size : index) ->
+    tensor<?x24x48xf32> {
+  %c0 = arith.constant 0 : index
+  %0 = tensor.insert_slice %arg1 into %arg0[%offset, 2, 0] [1, %size, 48] [1, 1, 1] :
+      tensor<1x?x48xf32> into tensor<?x24x48xf32>
+  return %0 : tensor<?x24x48xf32>
+}
+// CHECK-LABEL: func.func @insert_slice_convert
+//  CHECK-SAME:   %[[ARG0:[a-zA-Z0-9_]+]]
```

Comment:
if capturing named things it's a good idea to name them - so %arg0 should be %target_tensor / %[[TARGET_TENSOR]], etc